templates/terraform/examples/app_engine_service_split_traffic.tf.erb

```diff
@@ -0,0 +1,69 @@
+resource "google_storage_bucket" "bucket" {
+	name = "<%= ctx[:vars]['bucket_name'] %>"
+}
+
+resource "google_storage_bucket_object" "object" {
+	name   = "hello-world.zip"
+	bucket = "${google_storage_bucket.bucket.name}"
+	source = "./test-fixtures/appengine/hello-world.zip"
+}
+
+resource "google_app_engine_standard_app_version" "myapp_v1" {
+  version_id = "v1"
+  service = "myapp"
+  runtime = "nodejs10"
+  noop_on_destroy = true
+  entrypoint {
+    shell = "node ./app.js"
+  }
+  deployment {
+    zip {
+      source_url = "https://storage.googleapis.com/${google_storage_bucket.bucket.name}/hello-world.zip"
+    }  
+  }
+  env_variables = {
+    port = "8080"
+  } 
+  depends_on = ["google_storage_bucket_object.object"]
+
+}
+resource "google_app_engine_standard_app_version" "myapp_v2" {
+  version_id = "v2"
+  service = "myapp"
+  runtime = "nodejs10"
+  entrypoint {
+    shell = "node ./app.js"
+  }
+  deployment {
+    zip {
+      source_url = "https://storage.googleapis.com/${google_storage_bucket.bucket.name}/hello-world.zip"
+    }  
+  }
+  env_variables = {
+    port = "8080"
+  } 
+  depends_on = ["google_app_engine_standard_app_version.myapp_v1"]
+}
+
+resource "google_app_engine_service_split_traffic" "myapp" {
+  service_id = "${google_app_engine_standard_app_version.myapp_v2.service}"
+  migrate_traffic = false
+  split {
+    shard_by = "IP"
+    allocations = {
+      v1 = 0.75
+      v2 = 0.25
+    }
+  }
+  depends_on = ["google_app_engine_standard_app_version.myapp_v2"]
```

Comment:
This is redundant with the interpolation in `service_id` I think, and can be removed.

Reply:
This is needed in order to get the integration tests passed; as the test fail because it can not destroy the service which has zero traffic.

Reply:
I think the same connection between resources is established by https://github.com/GoogleCloudPlatform/magic-modules/pull/2269/files#diff-458f601ca7d7a40a89cb7b8bf8595b19R49, though?

Reply:
@rileykarson 
So I tested it , here is what is happening. We create v1 it takes 100% of the traffic, we create v2 it has no traffic, we use split traffic resource to create 75-25% split between the two versions.
Now when the destroy happens, it tries to destroy v1 which is having 75% traffic, and it fails as the v1 is part of split traffic, I don't think it automatically transfers the traffic to other v1.

What happens when I add the second split, it migrate all the traffic to v2, then destroy works fine.